<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPresentationParameters</class>
 <widget class="QWidget" name="WidgetPresentationParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>LAB_FIELD</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxField">
     <item>
      <property name="text">
       <string>LAB_EUCLIDEAN_NORM</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelTimeStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>LAB_TIMESTEP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxTimeStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelMinVal">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>LAB_MIN_VALUE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMinVal">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelMaxVal">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>LAB_MAX_VALUE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxVal">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>LAB_COLOR_MAP</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboBoxColorMap">
     <item>
      <property name="text">
       <string>LAB_BLUE_TO_RED</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LAB_COOL_TO_WARM</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBoxScalarBarRange">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>LAB_ALL_TIMESTEPS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LAB_ONE_TIMESTEP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LAB_USER_RANGE</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>LAB_SCALARBAR</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>LAB_RANGE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
